elastic/kibana
# [Dataset quality] Added malformed docs column to table

Closes https://github.com/elastic/kibana/issues/170220.

### Changes
- New endpoint added to query malformed docs in elasticsearch `GET /internal/dataset_quality/data_streams/malformed_docs` 
- Decoded response from apis in `data_streams_stats_client.ts` as suggested by @tonyghiani in https://github.com/elastic/kibana/pull/171777.
- New synthtrace scenario, malformed logs, where we ingest documents that will have `_ignored` properties.
- Malformed Docs column was added to `columns.tsx`.

#### Demo

https://github.com/elastic/kibana/assets/1313018/07a76f13-a837-4621-9366-63053a51b489

### How to test?
1. Go to https://yngrdyn-deploy-kiban-pr172462.kb.us-west2.gcp.elastic-cloud.com/app/observability-log-explorer/dataset-quality
2. `Malformed docs` column should be present and should be sortable
